### PR TITLE
Fix Java warnings for com.ibm.ws.junit.extensions

### PR DESCRIPTION
**Pull request by chackoj**
#build

### CODE REVIEW

File: dev/com.ibm.ws.junit.extensions/src/test/common/TestFile.java [hunk 1]
```diff
@@ -86,31 +86,32 @@ public static void unzip(File sourceZip, File targetDir) throws Exception {
...
+                    targetFile.mkdir();
+                } else {
+                    System.out.println("Unzipping: " + ze.getName() + " into " + targetFile);
+
+                    byte[] buffer = new byte[2048];
+                    BufferedInputStream bis = null;
+                    BufferedOutputStream bos = null;
+
+                    try {
+                        bis = new BufferedInputStream(zf.getInputStream(ze));
```
Comment (ngmr) [inline dev/com.ibm.ws.junit.extensions/src/test/common/TestFile.java:104]: 'bis' and 'bos' should be in try-with-resources, sjouldn't they?

File: dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java [hunk 2]
```diff
@@ -61,11 +61,6 @@ public void run(RunNotifier notifier) {
                         numTests++;
                     }
                 }
-            } finally {
-                try {
-                    reader.close();
-                } catch (IOException e) {
-                }
             }
```
Comment (ngmr) [inline dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java:64]: The previous code swallowed any IOException from the close(); the changed code allows it to propagate.
Did you intend to make this behaviour change ?

Comment (chackoj) [reply dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java:57]: No. Good spot — I will fix this.

**Comment by chackoj** [comment]
#build

**Comment by chackoj** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-
CD#action=com.ibm.team.build.viewResult&id=_KtgqsAqFEeuIgpJAZoWiKQ (all tests)

**Comment by chackoj** [comment]
#run-libby-bot

File: dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java (hunk 2 above)
Comment (chackoj) [reply dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java:57]: Fixed

**Review by ngmr — APPROVED** [review]
Looks okay to me

**Comment by ngmr** [comment]
#run-libby-bot

**Comment by ngmr** [comment]
#build

**Pull request merged**